<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HighlighterDialog</class>
 <widget class="QDialog" name="HighlighterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Colors and Font Property for Text Highlighting</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="color_buttons"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>209</x>
     <y>9</y>
     <width>91</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="italic_buttons"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>131</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="weight_boxes"/>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
